features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Listing/CommonListing.jsx

```diff
@@ -308,12 +323,12 @@ class CommonListing extends React.Component {
                     {active && allTags && allTags.length > 0 && <ApiBreadcrumbs selectedTag={selectedTag} />}
                     <div className={classes.listContentWrapper}>
                         {listType === 'grid' && (
-                            <ApiContext.Provider value={{ apiType }}>
+                            <ApiContext.Provider value={{ apiType, isMonetizationEnabled }}>
```

Comment:
Why are we creating a new context here again?

Reply:
API context was already created previously, i only needed to add monetization enabled value to the API context. Now in the API Context API type and isMonetizationEnabled attributes are available.